content/release-notes/1.12.0.md

```diff
@@ -0,0 +1,38 @@
+---
+date: 2020-01-31T19:55:39-08:00
+linktitle: "1.12.0"
+title: "1.12.0"
+---
+
+{{<features>}}
+* Namespace can be specified for Helm charts.
+* kurl.sh supports CentOS 7.7
+* Node resource analyzers
+{{</features>}}
+
+{{<changes>}}
+* If license is updated during a sync, a dialog is shown linking to version history.
+* Better feedback when saving app config.
+* Support bundle analysis now has breadcrumbs.
+* `replicated/kurl-util` container uses a versioned tag instead of `latest`.
+* `kots` CLI now includes an `Authorization` header when making requests to `kotsadm-api`.
+* Airgap upload endpoints now require `Authorization` headers.
+* Helm chart rendering errors are now displayed in the Admin Console.
+* The Dashboard link has been removed from the license upload pages.
+* License is synced to the latest version on installation.
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
+{{</changes>}}
+
+{{<fixes>}}
+* Airgap upload no longer tries to authenticate with docker registry.
```

Comment:
Why not? What happened when it tried to authenticate with docker registry?

Reply:
It's airgap.  There is no internet access.

Reply:
I understand, but you can access the internet in some airgap installs. I've used the docker registry in an airgap install with no problem.  Is this no longer possible?